libs/ui-lib/lib/ocm/components/clusterConfiguration/platformIntegration/ExternalPlatformDropdown.tsx

```diff
@@ -57,7 +57,12 @@ const getDisabledReasonForExternalPlatform = (
       cpuArchitecture,
     );
   } else if (platform === 'nutanix' || platform === 'vsphere') {
-    return `${ExternalPlatformLabels[platform]} integration is not supported for Single-Node OpenShift`;
+    return `${ExternalPlatformLabels[platform]} integration is not supported for Single-Node OpenShift.`;
+  } else if (
+    cpuArchitecture === CpuArchitecture.ppc64le ||
+    cpuArchitecture === CpuArchitecture.s390x
+  ) {
```

Comment:
shouldn't we get this from the BE?

Reply:
no, because is a combination of SNO+cpu architecture